package.json

```diff
@@ -30,7 +30,7 @@
   },
   "dependencies": {
     "bootstrap": "^3.3.5",
-    "bootstrap3":"3.3.5",
+    "bootstrap3": "^3.3.5",
```

Comment:
You have duplicate bootstrap dependencies,  consider removing one

Reply:
Thanks for pointing that out..I'll remove them in the subsequent code push.Please let me notify you when I do.